src/wallet-user/transaction-protection/LocsHistory.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { useCommonContext } from '../../common/CommonContext';
+import Table, { Cell, EmptyTableMessage, DateCell } from '../../common/Table';
+import LocStatusCell from '../../common/LocStatusCell';
+import LocIdCell from '../../common/LocIdCell';
+
+export default function TokenizationRequestsHistory() {
+    const { locRequestsHistory } = useCommonContext();
+
+    if(locRequestsHistory === null) {
+        return null;
+    }
+
+    return (
+        <Table
+            columns={[
```

Comment:
Sam remark here: If we don't show the reject reason to the user, what's the point to ask the LO to provide one ?

Reply:
Very true.